src/Tribe/RSVP.php

```diff
@@ -375,32 +375,32 @@ public function update_attendee_data( $data, $order_id, $event_id ) {
 
 		// Get the Attendee Data, it's important for testing
 		foreach ( $rsvp_orders as $test_attendee ) {
-			if ( $order_id !== $test_attendee['order_id'] ) {
+			if ( $order_id !== $test_attendee[ 'order_id' ] ) {
 				continue;
 			}
 
 			$attendee = $test_attendee;
 		}
 
 		// Dont try to Save if it's restricted
-		if ( ! isset( $attendee['product_id'] )
-		     || $this->tickets_view->is_rsvp_restricted( $event_id, $attendee['product_id'] )
+		if ( ! isset( $attendee[ 'product_id' ] )
+		     || $this->tickets_view->is_rsvp_restricted( $event_id, $attendee[ 'product_id' ] )
```

Comment:
no spaces around the array key unless the key name is a variable — `$attendee['product_id']` is fine here.